src/content/pages/who-we-are.mdx

```diff
@@ -26,15 +26,15 @@ compete in one or more team sports outside of Scouting as well!
 
 ### Weekly Meetings
 
-Troop 150 meets weekly on Wednesdays 7 - 8:30PM at St. Andrews Church: [3228 SW Sunset Blvd, Portland, OR 97239](https://www.google.com/maps/place/St+Andrew's+Presbyterian+Church/@45.4885215,-122.7100107,15z/data=!4m6!3m5!1s0x54950bb10d0ba7af:0xaed6dbdca741bfaa!8m2!3d45.4885215!4d-122.7100107!16s%2Fg%2F1td7bw8k?entry=ttu&g_ep=EgoyMDI0MDkxNS4wIKXMDSoASAFQAw%3D%3D).
+Troop 150 meets weekly on Wednesdays 7 - 8:30PM at St. Andrews Church: <a href="https://www.google.com/maps/place/St+Andrew's+Presbyterian+Church/@45.4885215,-122.7100107,15z/data=!4m6!3m5!1s0x54950bb10d0ba7af:0xaed6dbdca741bfaa!8m2!3d45.4885215!4d-122.7100107!16s%2Fg%2F1td7bw8k?entry=ttu&g_ep=EgoyMDI0MDkxNS4wIKXMDSoASAFQAw%3D%3D">3228 SW Sunset Blvd, Portland, OR 97239.</a>
```

Comment:
Nice! To make this open in a new tab, you'll want to add an attribute to the a tag: `target="_blank"`.